src/sdk/sql_cluster_router.cc

```diff
@@ -363,27 +363,23 @@ bool SQLClusterRouter::GetMultiRowInsertInfo(const std::string& db, const std::s
                                              std::vector<uint32_t>* str_lengths) {
     if (status == NULL || table_info == NULL || default_maps == NULL || str_lengths == NULL) {
         status->msg = "insert info is null";
-        LOG(WARNING) << status->msg;
```

Comment:
If no warning log, it's hard to debug. We have some confused msgs, e.g. `"can't find column " + col_name + " in table "`, `col_name` is a distraction. May be we can make a func or macro, e.g. 
```
RETURN_WITH_WARN(code, msg):
    status.msg = msg
    log(warn) << msg
    return status

```

Reply:
we can do this in #656.